addons/source-python/packages/source-python/translations/strings.py

```diff
@@ -217,12 +217,6 @@ def _replace_escaped_sequences(given_string):
         # Return the replaced string
         return given_string
 
-    def get_strings(self, key, **tokens):
```

Comment:
This method was added because of PR https://github.com/Source-Python-Dev-Team/Source.Python/pull/43.